cli/azd/cmd/package.go

```diff
@@ -96,7 +98,7 @@ type PackageResult struct {
 func (pa *packageAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 	// Command title
 	pa.console.MessageUxItem(ctx, &ux.MessageTitle{
-		Title: "Packaging services (azd package)",
+		Title: fmt.Sprintf("Packaging services (azd package) %s", output.WithWarningFormat("(BETA)")),
```

Comment:
I think this will look a little out-of-place when `package` is called from `up`. Should we make the BETA formatting only if it runs as the command?

Reply:
The command is running - so I feel it is consistent with the rest of the messaging.  We're just calling a beta feature from a standard GA flow.